### ISSUE TITLE
1.3rc2: Installation issue

### ISSUE THREAD

**Issue by dmr** [pygraphviz/pygraphviz#66]
python 3.4.3 on OSX: `pip install pygraphviz==1.3rc2`

causes `TypeError: Can't convert 'bytes' object to str implicitly`: somewhere around here:

```
    creating build/temp.macosx-10.10-x86_64-3.4
    creating build/temp.macosx-10.10-x86_64-3.4/pygraphviz
    clang -Wno-unused-result -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -Ib'/usr/local/Cellar/graphviz/2.38.0/include/graphviz' -I/usr/local/Cellar/python3/3.4.3_2/Frameworks/Python.framework/Versions/3.4/include/python3.4m -c pygraphviz/graphviz_wrap.c -o build/temp.macosx-10.10-x86_64-3.4/pygraphviz/graphviz_wrap.o
    pygraphviz/graphviz_wrap.c:3321:12: warning: incompatible pointer to integer conversion returning 'Agsym_t *' (aka 'struct Agsym_s *') from a function with result type 'int' [-Wint-conversion]
        return agattr(g, kind, name, val);
               ^~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by hagberg** [comment]
Could you post the entire error and your version of graphviz?

**Comment by mmarkov** [comment]
I believe the source of issue is here `-Ib'/usr/local/Cellar/graphviz/2.38.0/include/graphviz'` 
and I fixed it in #32. Could you please try to install latest pygraphviz directly from git using `pip install git+git://github.com/pygraphviz/pygraphviz.git`

**Comment by dmr** [comment]
`pip install git+git://github.com/pygraphviz/pygraphviz.git`works perfectly fine, thanks!
